src/main/java/goodgenerator/loader/ComponentAssemblyLineRecipeLoader.java

```diff
@@ -139,11 +159,13 @@ private static void generateAsslineRecipes() {
                         }
                     }
                     fixedInputs = compactItems(fixedInputs, info.getRight());
-                    replaceIntoFluids(fixedInputs, fixedFluids, 128);
+                    replaceIntoFluids(fixedInputs, fixedFluids, 64);
```

Comment:
can you make the 64 into a constant? I can't understand what it means from looking at it here. and if you want can do the other ones too.